### PR TITLE
[semver:minor] Add support for container-secrets-updates w/ fixed shellcheck

### LINKED ISSUE #123: feat: container-secrets-updates for ECS Task Definition
## Describe Request:
Similar to [container-env-var-updates](https://github.com/CircleCI-Public/aws-ecs-orb/blob/8f17662a21ee91a7b217897666b199a7d0bb5e14/src/commands/update-task-definition.yml#L23-L34), Use this to update or set the references of secret variables that will be defined for the containers. The supported references are either the full ARN of the AWS Secrets Manager secret or the full ARN of the parameter in the AWS Systems Manager Parameter Store

https://github.com/CircleCI-Public/aws-ecs-orb/pull/27/files was trying to address it, but it was rejected because of not separating "environment" and "secrets"

## Examples:
```
container-secrets-updates:
  container=<container-name>,name=<env-var-name>,valueFrom=<env-var-value>,container=...,name=...,valueFrom=...,
```

## Supporting Documentation Links:
https://docs.aws.amazon.com/AmazonECS/latest/userguide/task_definition_parameters.html

### PR DESCRIPTION
**Pull request by jazwiecki**
Based on https://github.com/CircleCI-Public/aws-ecs-orb/pull/124, this should fix the shellcheck so we can get this feature in the orb to fix #123

### CODE REVIEW

File: src/scripts/update-bluegreen-service-via-task-def.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ echo "Created CodeDeploy deployment: $DEPLOYMENT_ID"
 
 if [ "$ECS_PARAM_VERIFY_REV_DEPLOY" == "1" ]; then
     echo "Waiting for deployment to succeed."
-    if $(aws deploy wait deployment-successful --deployment-id "${DEPLOYMENT_ID}"); then
+    if aws deploy wait deployment-successful --deployment-id "${DEPLOYMENT_ID}" then
```
Comment (KyleTryon) [inline src/scripts/update-bluegreen-service-via-task-def.sh:21]: before the `then` I am fairly certain a semicolon will be needed. 
image

**Review by KyleTryon — CHANGES_REQUESTED** [review]
Thank you for the help! I think there is one tiny change needed, mentioned below.

**Comment by jazwiecki** [comment]
@KyleTryon made the change, can you re-review when you have a moment?

**Comment by jazwiecki** [comment]
@KyleTryon not to be a pest but I think this should be approvable

**Comment by brivu** [comment]
These shell check errors were addressed in PR #153

**Pull request closed without merging**